_includes/v21.2/sidebar-data/reference.json

```diff
@@ -1344,35 +1344,6 @@
           "/${VERSION}/cluster-settings.html"
         ]
       },
-      {
-        "title": "Performance Optimization",
```

Comment:
Seems odd to me that these topics, which are core to CockroachDB's architecture/design, would not also be mentioned in `Reference`. Is there an objection to putting them in both `Develop` and `Reference`?